packages/web-components/src/link/link.options.ts

```diff
@@ -0,0 +1,44 @@
+import { AnchorAttributes, AnchorTarget } from '../anchor-button/anchor-button.options.js';
+import type { ValuesOf } from '../utils/index.js';
+
+/**
+ * Link Appearance constants
+ * @public
+ */
+export const LinkAppearance = {
+  subtle: 'subtle',
+} as const;
+
+/**
+ * An Link can be secondary, primary, outline, subtle, transparent
```

Comment:
Is this comment accurate? Looks like `LinkAppearance` above only has `subtle` as an option.